_includes/setup-steps.html

```diff
@@ -1,13 +1,13 @@
 <section>
     <h3 class="c-subheading-2">Connecting to a network</h3>
     <blockquote class="c-blockquote">
-    <p class="c-paragraph">NOTE: Skip this step if you've already connected your device to a network in Step 1.</p>
+    <p class="c-paragraph">NOTE:  Skip this step if you've already connected your device to a network in Step 1.</p>
```

Comment:
Is the extra space intentional ?

Reply:
Yes - it looks strange without an extra space on the site :(